typings/Serverless/index.d.ts

```diff
@@ -35,7 +35,9 @@ declare namespace Serverless {
     extraServicePath?: string;
   }
 
-  type Event = { [type: string]: { rate: string; input?: object } };
+  type Event = {
+    schedule: { rate: string; input?: object } | string;
```

Comment:
there's also an enabled field which could be used and respected in the code if enabled: false